airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -138,6 +138,7 @@
     "EventWhoRelation",
     "FieldSecurityClassification",
     "KnowledgeArticle",
+    "KnowledgeArticleVersion",
```

Comment:
https://salesforce.stackexchange.com/a/71335 this answer suggests we should add the following entities too: 

```suggestion
    "KnowledgeArticleVersion",
    "KnowledgeArticleVersionHistory",
    "KnowledgeArticleViewStat",
    "KnowledgeArticleVoteStat",
```

Reply:
ooh, good call. 

It looks like we're still missing these:
* List item (not sure if this is the name of an endpoint or some generic type of API object)
* LeadStatus
* OpportunityStage
* UserRecordAccess

For the last three, we don't yet support those streams - do you want to just add them in preemptively, or leave them until the streams are actually implemented?

Reply:
happy to add them pre-emptively!